dev/src/index.html

```diff
@@ -1,32 +1,390 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <title>Botfront - Web Chat</title>
-  <script>
+<html>
+  <head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+  </head>
+  <body>
+    <style>
```

Comment:
@MattiaSarti  Can we create a CSS and js file and import it here?

Reply:
Done! It's still working with the refactored code:
![immagine](https://github.com/user-attachments/assets/7f80e04b-01bf-4c3f-9eda-c87700438cf1)

Reply:
PS: [here's the reference for Webpack's configuration changes in my commit](https://github.com/webpack/webpack/discussions/18049)